src/pages/Article/ArticlePage.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PageLayout from '../../components/PageLayout';
+import { Button, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import PropTypes from 'prop-types';
+import { useTheme } from '@mui/material/styles';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+
+function ArticlePage(props) {
+  const theme = useTheme();
+  return (
+    <PageLayout viewportPage title='Article'>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+          height: 'calc(100% - 60px)',
+          overflowY: 'auto',
+        }}
+      >
+        <Box
+          sx={{
+            height: '80%',
+            width: '60%',
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'center',
+            alignItems: 'center',
+            backgroundColor: theme.palette.primary.light ,
+            padding: '20px',
```

Comment:
Dont work with px values here.
We have defined padding values in our UI Guidelines. These are the Default MUI Values so you can use them here directly https://mui.com/system/spacing/
<img width="615" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/df4910df-97d3-4273-bf28-bc908b76c5a0">